pipeline/scripts/ci/getPipelineStages.py

```diff
@@ -65,12 +65,18 @@ def fetch_stages(args):
     current_dir = os.path.dirname(os.path.abspath(__file__))
     metadata_dir = os.path.abspath(f"{current_dir}/../../metadata")
     metadata_file = f"{metadata_dir}/{args['rhcephVersion']}.yaml"
-    data = yaml.safe_load(open(metadata_file, "r"))
+    metadata_content = yaml.safe_load(open(metadata_file, "r"))
+    metadata_overrides = metadata_content.get("overrides")
+    ibmc_overrides = metadata_overrides.pop("ibmc", None)
+    openstack_overrides = metadata_overrides.pop("openstack", None)
```

Comment:
`metadata_overrides = None` would lead to `NoneType has to attribute pop` if there is no key/value `overrides` in the meta file. Either set
`metadata_overrides = metadata_content.get("overrides", {})`

Reply:
Sure will make this change and raise a PR.

Reply:
https://github.com/red-hat-storage/cephci/pull/1925 - PR created for this change